cli/ballerina-cli/src/main/java/io/ballerina/cli/launcher/Main.java

```diff
@@ -462,13 +472,23 @@ private String promptForInput(String msg) {
         @CommandLine.Option(names = { "--version", "-v" }, hidden = true)
         private boolean versionFlag;
 
+        @CommandLine.Parameters
+        private List<String> argList = new ArrayList<>();
```

Comment:
We only need one arg here, isnt it? If so, we can restrict the limit using `arity`. Refer https://picocli.info/#_arity